internal/cli/cmd/apply/apply.go

```diff
@@ -17,6 +17,7 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/efficientgo/core/merrors"
```

Comment:
while doing something in prometheus go client, found this lib wrote by our dear bartek, efficient and lighter than multierror